.github/workflows/template-build.yml

```diff
@@ -94,6 +94,7 @@ jobs:
           NEXT_PUBLIC_TINA_BRANCH: ${{ github.head_ref || github.ref_name }}
           NEXT_PUBLIC_ZENDESK_CHAT_KEY: ${{ secrets.NEXT_PUBLIC_ZENDESK_CHAT_KEY }}
           GOOGLE_RECAPTCHA_KEY: ${{ secrets.GOOGLE_RECAPTCHA_KEY }}
+          GOOGLE_RECAPTCHA_SITE_KEY: ${{ secrets.GOOGLE_RECAPTCHA_SITE_KEY }}
```

Comment:
given that this is now stored in kv, can we source the secret values from there instead of using github secrets?

Reply:
We need to log in before the build and then pass the kv secret value to it, is it a right way doing it?

Reply:
done